Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -3194,7 +3193,7 @@ void HTMLMediaElement::setReadyState(MediaPlayer::ReadyState state)
 
     m_tracksAreReady = tracksAreReady;
 
-    ALWAYS_LOG(LOGIDENTIFIER, "new state = ", state, ", current state = ", m_readyState);
+    HTMLMEDIAELEMENT_RELEASE_LOG(HTMLMEDIAELEMENT_SETREADYSTATE, convertEnumerationToString(state).utf8().data(), convertEnumerationToString(m_readyState).utf8().data());
```

Comment:
Ideally, we would do the conversion if we are actually logging.
`ALWAYS_LOG  / Logger::logAlwaysVerbose` will do the conversion only if `willLog` will return true.
We are losing some efficiency here.